zebra/zebra_evpn_mac.c

```diff
@@ -853,6 +903,29 @@ void zebra_evpn_print_mac_hash_detail(struct hash_bucket *bucket, void *ctxt)
 	zebra_evpn_print_mac(mac, vty, json_mac_hdr);
 }
 
+#define MACIP_BUF_SIZE 128
+static char *zebra_evpn_dump_macip_flags(uint8_t flags, char *buf, size_t len)
+{
+	if (flags == 0) {
+		snprintfrr(buf, len, "None ");
+		return buf;
+	}
+
+	snprintfrr(
+		buf, len, "%s%s%s%s%s%s%s",
+		CHECK_FLAG(flags, ZEBRA_MACIP_TYPE_STICKY) ? "Sticky MAC " : "",
```

Comment:
This set of flags comes from zclient.h - so maybe the outputter should be available from zclient.c? And could there be a note explaining that the outputter exists, and needs to be maintained?

Reply:
This is a good point - these flags are also displayed in bgp_evpn.c and so having the utility in lib would enable its use universally.

Reply:
fixed